scripts/create-rpms.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#create source tarball
+rm -rf ../build
+mkdir ../build
+cd ../build
+BUILD_DIR=${PWD}
+tar czvf opae.tar.gz ../../opae-sdk --transform s/opae-sdk/opae/
+cp opae*.tar.gz ~/rpmbuild/SOURCES/
+cp ../opae-sdk-rpm.spec ~/rpmbuild/SPECS/
+
+cd ~/rpmbuild/SPECS/
+
+#clean RPMS directory
+rm -rf ../RPMS/x86_64/opae-*
+
+#generate RPMS
+rpmbuild -ba opae-sdk-rpm.spec
+
+#copy RPMS to build directory
+cp ~/rpmbuild/RPMS/x86_64/opae-* $BUILD_DIR/
```

Comment:
add rpmlint  and mock 
 rpmlint hello.spec ../SRPMS/hello* ../RPMS/*/hello*
 mock --verbose ../SRPMS/hello-2.10-1.fc25.src.rpm

Reply:
I feel like these steps could be part of test rather than each build.